test/common/cdp.py

```diff
@@ -14,7 +14,6 @@
 
 TEST_DIR = os.path.normpath(os.path.dirname(os.path.realpath(os.path.join(__file__, ".."))))
 
-
```

Comment:
Two empty lines is actually correct as per PEP-8. We aren't very consistent in that in Cockpit, and we don't have pycodestyle in our tests, but just saying.

Reply:
I Think this got introduced by accident, sorry!